lib/stream/xlsx/worksheet-writer.js

```diff
@@ -243,9 +246,10 @@ class WorksheetWriter {
     this._writeSheetProtection();
     this._writePageMargins();
     this._writePageSetup();
-    this._writeBackground();
     this._writeHeaderFooter();
     this._writeRowBreaks();
+    this._writeDrawings(); // Note: must be after rowBreaks
+    this._writeBackground(); // Note: must be after drawing
```

Comment:
this order change is needed to use both addImage and addPageBreak same as https://github.com/exceljs/exceljs/pull/590